### PR TITLE
UI tweaks of the physics doc link widget

### PR DESCRIPTION
**Pull request by amzn-sean**
Signed-off-by: amzn-sean <75276488+amzn-sean@users.noreply.github.com>

image

### CODE REVIEW

File: Gems/PhysX/Code/Editor/DocumentationLinkWidget.cpp [hunk 1]
```diff
@@ -21,8 +21,9 @@ namespace PhysX
             setTextInteractionFlags(Qt::TextBrowserInteraction);
             setOpenExternalLinks(true);
             setAlignment(Qt::AlignCenter);
-            setContentsMargins(60, 15, 60, 15);
+            setContentsMargins(60, 7, 60, 7);
             setWordWrap(true);
+            setStyleSheet(QString::fromUtf8("background-color: rgb(51, 51, 51);"));
```
Comment (greerdv) [inline Gems/PhysX/Code/Editor/DocumentationLinkWidget.cpp:26]: It feels like there ought to be a way to do this with a variable - what happens if we ever want to change the colour scheme, or add different colour scheme options? Is this what the colour settings in the rest of the engine look like?

Comment (amzn-sean) [reply Gems/PhysX/Code/Editor/DocumentationLinkWidget.cpp:26]: i was looking for something like that, but it appears this is how it was done. if you know of a common colour variable i'm all ears!

Comment (greerdv) [reply Gems/PhysX/Code/Editor/DocumentationLinkWidget.cpp:26]: I can't find it anywhere, so I guess just have to go with this

**Review by greerdv — APPROVED** [review]

File: Gems/PhysX/Code/Editor/DocumentationLinkWidget.cpp (hunk 1 above)
Comment (amzn-sean) [reply Gems/PhysX/Code/Editor/DocumentationLinkWidget.cpp:26]: i made this feature request https://github.com/o3de/o3de/issues/2260 to improve this in the future

**Review by jonawals — APPROVED** [review]

**Review by moraaar — APPROVED** [review]

**Pull request merged**